ldi-core/ngsiv2-to-ld/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/services/NgsiV2ToLdTranslatorService.java

```diff
@@ -0,0 +1,48 @@
+package be.vlaanderen.informatievlaanderen.ldes.ldi.services;
+
+import be.vlaanderen.informatievlaanderen.ldes.ldi.exceptions.InvalidNgsiLdContextException;
+import be.vlaanderen.informatievlaanderen.ldes.ldi.valuobjects.LinkedDataModel;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class NgsiV2ToLdTranslatorService {
```

Comment:
Would propose to move this class to its parent package (be.vlaanderen.informatievlaanderen.ldes.ldi)
Since this is the main service that combines everything.